core/src/main/kotlin/org/evomaster/core/problem/rest/service/fitness/AbstractRestFitness.kt

```diff
@@ -1196,6 +1197,35 @@ abstract class AbstractRestFitness : HttpWsFitness<RestIndividual>() {
         }
     }
 
+    private fun handleForgottenAuthentication(
+        individual: RestIndividual,
+        actionResults: List<ActionResult>,
+        fv: FitnessValue
+    ) {
+        val getPaths = individual.seeMainExecutableActions()
```

Comment:
should be on the endpoints, not paths. you can get it with the `getName`, which for REST actions is the endpoint